docs/sink/flink-connector-oceanbase-tools-cdc_cn.md

```diff
@@ -0,0 +1,188 @@
+# Flink Connector OceanBase By Tools CDC
+
+[English](flink-connector-oceanbase-tools-cdc.md) | 简体中文
+
+本项目是一个可以支持通过Flink命令行构建cdc任务同步到oceanbase的flink命令行工具，极大的简化了通过flink同步数据到oceanbase的命令书写。
+
+## 开始上手
+
+您可以在 [Releases 页面](https://github.com/oceanbase/flink-connector-oceanbase/releases) 或者 [Maven 中央仓库](https://central.sonatype.com/artifact/com.oceanbase/flink-connector-oceanbas-directload) 找到正式的发布版本。
+
+```xml
+<dependency>
+    <groupId>com.oceanbase</groupId>
+    <artifactId>flink-connector-oceanbase-tools-cdc</artifactId>
+    <version>${project.version}</version>
+</dependency>
+```
+
+如果你想要使用最新的快照版本，可以通过配置 Maven 快照仓库来指定：
+
+```xml
+<dependency>
+    <groupId>com.oceanbase</groupId>
+    <artifactId>flink-connector-oceanbase-tools-cdc</artifactId>
+    <version>${project.version}</version>
+</dependency>
+
+<repositories>
+    <repository>
+        <id>sonatype-snapshots</id>
+        <name>Sonatype Snapshot Repository</name>
+        <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url>
+        <snapshots>
+            <enabled>true</enabled>
+        </snapshots>
+    </repository>
+</repositories>
+```
+
+您也可以通过源码构建的方式获得程序包。
+
+```shell
+git clone https://github.com/oceanbase/flink-connector-oceanbase.git
+cd flink-connector-oceanbase
+mvn clean package -DskipTests
+```
+
+### 注意事项：
+
+* 目前项目支持使用Flink CDC接入多表或整库。同步时需要在 `$FLINK_HOME/lib` 目录下添加对应的 Flink CDC 依赖，比如 flink-sql-connector-mysql-cdc-\${version}.jar，flink-sql-connector-oracle-cdc-\${version}.jar ，flink-sql-connector-sqlserver-cdc-\${version}.jar
+* 依赖的 Flink CDC 版本需要在 3.1 以上，如果需使用 Flink CDC 同步 MySQL 和 Oracle，还需要在 `$FLINK_HOME/lib` 下增加相关的 JDBC 驱动。
+* 同步至oceanbase时，oceanbase的url连接串需要使用mysql的协议。
```

Comment:
Use `OceanBase` and `MySQL` as product names.

Reply:
Revised